samples/app/src/main/kotlin/com/bumble/appyx/app/MainActivity.kt

```diff
@@ -22,14 +23,15 @@ import com.bumble.appyx.core.modality.BuildContext
 class MainActivity : NodeActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
+        installSplashScreen()
```

Comment:
Should we use `installSplashScreen()`? I believe this is for legacy projects where you already have a splash screen. Is there not a dedicated splash screen activity in the library?

Reply:
https://developer.android.com/develop/ui/views/launch/splash-screen/migrate#remove_the_custom_splash_screen_activity

Reply:
If I don't do it the app crashes (at least on my device)
It's part of the step number 4 in [here](https://developer.android.com/develop/ui/views/launch/splash-screen/migrate#migrate_your_splash_screen_implementation)

Reply:
ah fair enough. I thought the API suggested that they had their own activity, and for developers to not use their own activity for this purpose.